cvat-ui/src/components/header/settings-modal/multiple-shortcuts-display.tsx

```diff
@@ -0,0 +1,169 @@
+    const selectRef = useRef<any>(null);
+    const [focus, setFocus] = useState(false);
+    const [pressedKeys, setPressedKeys] = useState<string[][]>([[]]);
+    const [currentIdx, setCurrentIdx] = useState<number>(0);
+    const [timer, setTimer] = useState<NodeJS.Timeout | null>(null);
+
+    useEffect(() => () => {
+        if (timer) clearTimeout(timer);
+    }, [timer]);
+
+    function unsetExistingShortcut(
+        conflictingShortcuts: Record<string, KeyMapItem>,
+        updatedSequence: string[],
+        shortcut: Record<string, KeyMapItem>): void {
+        const updatedShortcuts: Record<string, KeyMapItem> = {};
+        for (const [key, value] of Object.entries(conflictingShortcuts)) {
+            const currentItem: Record<string, KeyMapItem> = {
+                [key]: value,
+            };
+            if (isEqual(Object.keys(currentItem), Object.keys(shortcut))) {
+                const currentSequence = currentItem[Object.keys(shortcut)[0]].sequences;
+                const newSequence = updatedSequence.filter((s) => !currentSequence.includes(s))[0];
+                updatedShortcuts[key] = {
+                    ...value,
+                    sequences: [...currentSequence.filter((s) => !conflict(s, newSequence)), newSequence],
+                };
+            } else {
+                const commonSequence = updatedSequence.map((s) => {
+                    for (const seq of value.sequences) {
+                        if (conflict(seq, s)) {
+                            return seq;
+                        }
+                    }
+                    return '';
+                });
+                const newSequences = value.sequences.filter((s) => !commonSequence.includes(s));
+                updatedShortcuts[key] = { ...value, sequences: newSequences };
+            }
+        }
+        registerComponentShortcuts(updatedShortcuts);
+    }
+
+    function conflictNotifier(keyMapId: string, updatedSequence: string[]): void {
+        const shortcut = {
+            [keyMapId]: { ...keyMap[keyMapId], sequences: updatedSequence },
+        };
+        const conflictingShortcuts: Record<string, KeyMapItem> | null = conflictDetector(shortcut, keyMap);
+        if (conflictingShortcuts) {
+            Modal.confirm({
+                title: 'Shortcut conflict',
+                content: (
+                    <p>
+                        This sequence conflicts with the following shortcuts:
+                        <br />
+                        <strong>{Object.values(conflictingShortcuts)[0].name}</strong>
+                        {' '}
+                        in the scope
+                        {' '}
+                        <strong>
+                            {ShortcutScope[
+                                Object.values(conflictingShortcuts)[0].scope].split('_').join(' ')}
+                        </strong>
+                        <br />
+                        Would you like to unset the conflicting shortcuts?
```

Comment:
Do you think will it be possible to have conflict more than with one shortcut? 
Just from my understanding, if shortcut conflicts with two existing. Then before user change these two existing are already conflicted between each other (what is not possible to set)

Reply:
actually it is possible for two to not conflict with each other but conflict with a 3rd shortcut, example,
alt + c and alt + d are registered, these don't conflict with each other now if I set a new shortcut that is has the sequence alt, it will conflict with both.